trailblazer/server/api.py

```diff
@@ -111,7 +101,10 @@ def patch_analyses(analysis_service: AnalysisService = Provide[Container.analysi
     """Update data (such as status, visibility, comments etc.) for multiple analyses at once."""
     try:
         request_data = UpdateAnalyses.model_validate(request.json)
-        response: UpdateAnalysesResponse = analysis_service.update_analyses(request_data)
+        user: User = g.get("current_user")
+        response: UpdateAnalysesResponse = analysis_service.update_analyses(
+            data=request_data, user=user
```

Comment:
I'd consider passing down the user id, can't really give a good motivation for it though so take it with a grain of salt 😅 

I guess I don't like passing around database records in the api layer